### PR TITLE
Ballot Barry part 1

### PR DESCRIPTION
**Pull request by JeromeMartinez**
A couple of patches partially fixing [Barry Leiba's Discuss](https://mailarchive.ietf.org/arch/msg/cellar/rvI2_U8IydKtrOo1270wVOddyug/).

### CODE REVIEW

File: ffv1.md [hunk 1]
```diff
@@ -526,7 +526,7 @@ Figure: A pseudo-code description of the contexts of Range Non Binary Values. {#
 
 #### Initial Values for the Context Model
 
-At keyframes all Range coder state variables are set to their initial state.
+When `keyframe` (see (#frame)) value is 0, all Range coder state variables are set to their initial state.
```
Comment (dericed) [inline ffv1.md:529]: > When the `keyframe` value is 0 (see (#frame)),
 

and below

File: ffv1.md [hunk 2]
```diff
@@ -664,8 +664,8 @@ Figure: A pseudo-code description of the read of a signed integer in Golomb Rice
 |              |                                                         |
 |:-------------|:--------------------------------------------------------|
 |non ESC       | the k least significant bits MSB first                  |
-|ESC           | the value - 11, in MSB first order                      |
 
+|ESC           | the value - 11, in MSB first order                      |
```
Comment (dericed) [inline ffv1.md:668]: ?

File: ffv1.md [hunk 3]
```diff
@@ -104,23 +104,23 @@ Note: the operators and the order of precedence are the same as used in the C pr
...
+`a <= b`      is true when a is less than or equal b.
 
-`a == b`      means a is equal to b.
+`a == b`      is true when a is equal to b.
 
-`a != b`      means a is not equal to b.
+`a != b`      is true when a is not equal to b.
 
-`a && b`      means Boolean logical "and" of a and b.
+`a && b`      is true when Boolean logical "and" of a and b.
```
Comment (dericed) [inline ffv1.md:119]: The wording here is awkward. How about `is true when both a is true and b is true`?

File: ffv1.md [hunk 4]
```diff
@@ -104,23 +104,23 @@ Note: the operators and the order of precedence are the same as used in the C pr
...
 
-`a || b`      means Boolean logical "or" of a and b.
+`a || b`      is true when Boolean logical "or" of a and b.
```
Comment (dericed) [inline ffv1.md:121]: The wording here is awkward. How about `is true when either a is true or b is true`?

File: ffv1.md [hunk 5]
```diff
@@ -104,23 +104,23 @@ Note: the operators and the order of precedence are the same as used in the C pr
...
 
-`!a`          means Boolean logical "not" of a.
+`!a`          is true when Boolean logical "not" of a.
```
Comment (dericed) [inline ffv1.md:123]: The wording here is awkward. How about `is true when a is not true`?

File: ffv1.md (hunk 1 above)
Comment (JeromeMartinez) [reply ffv1.md:529]: All issues fixed but this one "and below", I don't get the expected meaning.

**Review by dericed — APPROVED** [review]

**Review by retokromer — APPROVED** [review]

**Pull request merged**